src/elemeno_ai_sdk/ml/automl/client.py

```diff
@@ -0,0 +1,86 @@
+import logging
+from typing import Dict
+
+import aiohttp
+
+
+class AutoMLClient:
+    def __init__(self, env: str, api_key: str) -> None:
+        self.env = env
+        self.api_key = api_key
+
+    @property
+    def base_url(self):
+        base_url = None
+        if self.env == "prod":
+            base_url = "https://c3po.ml.semantixhub.com"
+        elif self.env == "dev":
+            base_url = "https://c3po-stg.ml.semantixhub.com"
+        else:
+            raise ValueError("Invalid environment. Please use dev or prod.")
+        return base_url
+
+    @property
+    def headers(self):
+        return {"Content-Type": "application/json", "x-api-key": self.api_key}
+
+    async def list_jobs(self):
+        async with aiohttp.ClientSession(headers=self.headers) as session:
```

Comment:
These session blocks in `list_jobs`, `run_jobs` and `get_jobs` look the same, perhaps move them to a single function to improve readability?